AlpsSDK/ContextManager.swift

```diff
@@ -67,19 +68,11 @@ class ContextManager: NSObject, CLLocationManagerDelegate {
     // Update locations
     func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
         if let coord = locations.last {
-//            do {
-                //try self.onLocationUpdateClosure?(locations.last!)
-                //try alpsManager.updateLocation(latitude: coord.coordinate.latitude, longitude: coord.coordinate.longitude,
             self.onLocationUpdateClosure?(locations.last!)
             alpsManager.updateLocation(latitude: coord.coordinate.latitude, longitude: coord.coordinate.longitude,
-                                             altitude: coord.altitude, horizontalAccuracy: coord.horizontalAccuracy,
-                                             verticalAccuracy: coord.verticalAccuracy) {
-                    (_ location) in
-//                    NSLog("updating location to: \(coord.coordinate.latitude), \(coord.coordinate.longitude), \(coord.altitude)")
```

Comment:
We should be careful with the commentary swiftlint supress sometimes, because a lot of function that are not deprecated but just not working yet, are commented.

Reply:
This one was commented, but we use it to inform that location is updated.

Reply:
From my perspective - we can get rid of stuff that's not working.

On top we need much more control over logger. We can use `SwiftyBeaver` or write something on our own.

Reply:
Ok ! 
Let's start to use SwiftyBeaver. I like it 👍